__tests__/app.test.js

```diff
@@ -38,6 +38,75 @@ describe('/api/topics', () => {
 		});
 	});
 });
+describe('/api/articles', () => {
+	describe('GET', () => {
+		test('200: responds with an array of articles', () => {
+			return request(app)
+				.get('/api/articles')
+				.expect(200)
+				.then((response) => {
+					const allArticles = response.body.articles;
+					expect(Array.isArray(allArticles)).toBe(true);
+					expect(allArticles.length > 0).toBe(true);
+					allArticles.forEach((article) => {
+						expect(article).toHaveProperty('article_id', expect.any(Number));
+						expect(article).toHaveProperty('title', expect.any(String));
+						expect(article).toHaveProperty('topic', expect.any(String));
+						expect(article).toHaveProperty('author', expect.any(String));
+						expect(article).toHaveProperty('body', expect.any(String));
```

Comment:
There should be no body on the article for this endpoint. Imagine it like a blog. We don't see all the blog post on the home page :)